reversed.go

```diff
@@ -0,0 +1,19 @@
+package do
+
+import "reflect"
+
+/*
+Reversed returns the values of a given interface on a reversed basis.
+*/
+func Reversed(iterable interface{}) interface{} {
+	iterValue := getIterable(iterable)
+	returnValue := reflect.MakeSlice(iterValue.Type(), 0, 0)
```

Comment:
it is now being replaced with:

	returnValue := reflect.MakeSlice(iterValue.Type(), iterValue.Len(), 0)